### DISCUSSION TITLE
我因为太瘦所遇到的一些事情……

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#4498] [灌水]
![](https://c-ssl.dtstatic.com/uploads/item/201710/04/20171004174615_dNaAL.thumb.1000_0.jpeg)

# 如题，我是一个很瘦的男孩子，只有91斤………

---

男孩子（我）瘦的话感觉有好处也有坏处↓

### 好处：

1. 先天南梁圣体
2. 学校老师要求搬东西之类的事情基本不用管
3. 等待补充ing

### 坏处：

1. 被说细狗或者骷髅兵长（前者有点难受，后者勉勉强强还可以接受）
2. 容易被欺负
3. 可能会被你的同学左摸右摸，你还没什么办法……

---

未完待续……

**Comment by Bisaqulaor** [comment]
还是适当增加体重吧，对健康也好，少生病才是王道

**Comment by tingyu919** [comment]
去锻炼

**Comment by mztxy229** [comment]
我的肉分你一半，我是超重：（

**Comment by lllliufeng** [comment]
多吃点吧

**Comment by ggggNPC** [comment]
我大二94斤，除了被叫细狗，其余完全没有经历过啊😦
